src/theory/quantifiers/ematching/inst_match_generator.cpp

```diff
@@ -559,7 +563,7 @@ uint64_t InstMatchGenerator::addInstantiations(Node f,
   while (getNextMatch(f, m, qe, tparent) > 0)
   {
     if( !d_active_add ){
-      if (sendInstantiation(tparent, m))
+      if (sendInstantiation(tparent, m, InferenceId::UNKNOWN))
```

Comment:
What about this UNKNOWN?

Reply:
This is dead code that I believe will be deleted at some point.